tox.ini

```
@@ -77,6 +77,8 @@ deps =
     oldest: numpy==1.16.*
     oldest: scipy==1.2.*
 
+    all: git+https://github.com/ntessore/classy.git@v2.8.1#egg=classy
```

This has now been moved:
```suggestion
    all: git+https://github.com/skypyproject/classy.git@v2.8.1#egg=classy
```

---

The better way to specify the dependency is the following change to `setup.cfg`:
```diff
 [options.extras_require]
 test =
     pytest-astropy
 all =
     camb
+    classy @ git+https://github.com/skypyproject/classy.git@v2.8.1#egg=classy
 docs =
     sphinx-astropy
     matplotlib
```